src/network/mod.rs

```diff
@@ -183,8 +110,6 @@ impl SectionMap {
             }
         }
 
-        self.prune_neighbours();
```

Comment:
> pruning is now being called outside

This.

Reply:
I'd prefer call the pruning inside.
However, not a strong preference.
up to you.

Reply:
Can't do that, because we no longer have our section stored in this struct so we can't tell who is neighbour from inside it. Also I think we are going to address https://github.com/maidsafe/sn_routing/issues/2042 soon and this won't matter.